dropwizard-jetty/src/main/java/io/dropwizard/jetty/GzipHandlerFactory.java

```diff
@@ -213,15 +214,15 @@ public void setGzipCompatibleInflation(boolean gzipCompatibleInflation) {
     }
 
     /**
-     * @deprecated
+     * @deprecated excluding by user agent was removed in Jetty 10
```

Comment:
@rhowe We're still using Jetty 9.4.x because Jetty 10.x requires Java 11.

Reply:
I could've sworn I saw a deprecation warning from a linter somewhere but now I can't find it.

Since we know it's going away, do you think it should be marked as deprecated in order to signal the upcoming change? I have no strong opinion either way.

Reply:
@rhowe I marked it as deprecated in a previous PR because I already have an early build of Dropwizard with the Jakarta namespace running. But there are unstable tests at the `dropwizard-http2` module and I didn't want to raise a PR without knowing what's wrong there.

With that said: we're not far away from the ability to upgrade Jersey to 3.0.x, which I think is a good idea, since Jersey 3.1.0 should also be released soon.

Reply:
Cool, in that case I've amended this PR to add text to the `@deprecated` javadoc tags